lib/prod-server/app.js

```diff
@@ -30,6 +28,7 @@ module.exports = ({
   runner,
   staticDir
 }) => {
+  const server = connect()
```

Comment:
Is this ok, @bradparker ? 

Without moving the creating of the server, the server is a singleton and you can't test different server configurations.

I felt that if I called createApp, I would expect to create a different server each time. Having a server singleton would be an unexpected behavior (to me at least).

Reply:
It's great!